src/components/Clock/Clock.tsx

```diff
@@ -0,0 +1,58 @@
+import { Component } from 'react';
+
+type Props = {
+  clockName: string,
+};
+
+type State = {
+  date: Date,
+};
+
+const dateFormat = (value: Date) => value.toUTCString().slice(-12, -4);
+
+export class Clock extends Component<Props, State> {
+  state: Readonly<State> = {
+    date: new Date(),
+  };
+
+  timerId = 0;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ date: new Date() });
+
+      window.console.info(dateFormat(this.state.date));
```

Comment:
Don't use state here, it can be outdated
Save `new Date()` result to constant and reuse it for both setState and console.info